examples/firebase_example/.bazelrc

```diff
@@ -20,3 +20,6 @@ build --cxxopt='-std=gnu++14'
 # Firebase SPM support requires `-ObjC` linker option.
 # https://github.com/firebase/firebase-ios-sdk/blob/master/SwiftPackageManager.md#requirements
 build --linkopt='-ObjC'
+
+# Use sandbox for rules_xcodeproj to avoid duplicate definition errors.
+build:rules_xcodeproj --spawn_strategy=remote,worker,sandboxed,local
```

Comment:
I thought we had fixed this already? Or will @brentleyjones  PR fix the remaining edge cases

Reply:
I hope @brentleyjones fix will be the real fix. Once I fixed the Bazel binary issue, I ran into the duplicate definition error.